extensions/accessibilityExtension/README.md

```diff
@@ -0,0 +1,262 @@
+                            }
+                            },
+                        "accessibilityFeature": {
+                            "type": "string"
+                        },
+                        "accessibilityHazard": {
+                            "type": "string"
+                        },
+                        "url":{
+                            "type": "string",
+                            "format": "uri"
+                        },
+                    }
+                    "required": ["url","accessibilityFeature"],
+                    "additionalProperties": true
+                }
+            },
+            "additionalProperties": true
+        }
+    },
+    "additionalProperties": true
+}
+```
+
+```json
+// Accessibility OB 3.0 JSON Schema for Issuer Profile
+{
+    "$schema": "https://json-schema.org/draft/2019-09/schema#",
+    "$id": "https://openbadgespec.org/extensions/accessibilityExtension/schema_profile_obv3p0.json",
+    "type": "object",
+    "properties": {
+        "credentialSubject": {
+            "type": "object",
+            "properties": {
+                "issuer": {
+                    "type": "object",
+                    "properties": {
+                        "accessibilityAPI": {
+                            "type": "string"
+                        },
+                        "accessibilityControl": {
+                            "type": "array",
+                            "items": {
+                                "type": "string"
+                            }
+                            },
+                        "accessibilityFeature": {
+                            "type": "string"
+                        },
+                        "accessibilityHazard": {
+                            "type": "string"
+                        },
+                        "url":{
+                            "type": "string",
+                            "format": "uri"
+                        },
+                    },
+                    "required": ["url","accessibilityFeature"],
+                    "additionalProperties": true
+                }
+            },
+            "additionalProperties": true
+        }
+    },
+    "additionalProperties": true,
+}
+```
+
+A credential with this extension applied to both entities is shown below.
+It adds the url of the above JSON-LD context (assuming is
+`https://openbadgespec.org/extensions/accessibilityExtension/context-3.0.0.json`)
+in its `@context` declaration and the url of the JSON schemas in its
+`credentialSchema` attribute.
+
+> You only need to include the JSON schema of the extended entity to use.
+
+
+```json
+// Sample OpenBadgeCredential with Accessibility extension
+{
+    "@context": [
+        "https://www.w3.org/ns/credentials/v2",
+        "https://purl.imsglobal.org/spec/ob/v3p0/context-3.0.1.json",
+        "https://openbadgespec.org/extensions/accessibilityExtension/context-3.0.0.json"
```

Comment:
This URL doesn't work. Is it supposed to work before finalization of this PR?

Reply:
The URL will make sense when the PR is merged. So, included the contexts as files  with name `context-3.0.0.json` in every community extension with the goal to publish them to openbadgespec.org when the PR is merged